### PR TITLE
docs: improve content about accessing remote devices

### PR DESCRIPTION
**Pull request by NickIliev**

### CODE REVIEW

File: traffic/configure-android.md [hunk 1]
```diff
@@ -15,65 +15,57 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
...
 
-    1. On **Proxy**, select **Manual proxy**.
+    1. From the **Proxy** option, select **Manual proxy**.
 
-     - Enter the IP address of the Fiddler Everywhere host&mdash;for example, **192.168.148.39**.
+     - Enter the IP address of the Fiddler Everywhere host&mdash;for example, **192.168.100.50**.
 
      - Enter the Fiddler Everywhere proxy port. By default, the port is **8866**.
 
+     >important Ensure that all VPNs from the Fiddler host and the Android devices are turned off or bypassed.
```
Comment (rosen-vladimirov) [inline traffic/configure-android.md:51]: We need to check how to support this  - there are cases where devs will work on internal apps, which use APIs part of the internal network (will require VPN).

File: traffic/configure-android.md [hunk 2]
```diff
@@ -15,65 +15,57 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
...
 
-- Install [Fiddler Everywhere v. 1.0.0 and above](https://www.telerik.com/download/fiddler-everywhere).
-- Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine.
-Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
+- Install the latest version of [Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
 
-## Configuring Fiddler Everywhere
+- Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine. Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
+
+## Configuring Fiddler Everywhere Host
```
Comment (rosen-vladimirov) [inline traffic/configure-android.md:29]: We must note that Fiddler's root certificate must be trusted and HTTPS capturing to be enabled on the machine where Fiddler Everywhere is running. We are going to remove this limitation in the future, but we have it for the moment.

File: knowledge-base/how-to-capture-traffic-from-another-pc.md [hunk 3]
```diff
@@ -26,35 +26,42 @@ How to capture HTTP/HTTPS traffic from a client on the same network as the Fiddl
...
+- The Fiddler Everywhere host and the target using the Fiddler Everywhere proxy must be discoverable on the network. There must be no administrative limitations or security policies that prevent device discovery on the local network.
+
+    >tip If you can ping the Fiddler host from the target device and the other way around, that would indicate that both devices are "seeing" each other.
+
+- The Fiddler Everywhere host and the target using the Fiddler Everywhere proxy should have the Fiddler proxy port opened. By default, the port is **8866**.
+
+
+The requirement for using the same network is actually not a must. However, most remote networks, like those managed by ISPs, use firewalls and explicitly close most ports. That makes configuring and using Fiddler over the internet virtually impossible (as there are not many ISPs that will allow you to open a random port). If you are the administrator of two separate networks, you must ensure that the Fiddler proxy port is open and that both host and target machines are seeing each other (successful ping).
 
 ## Configuring Fiddler Everywhere (Host)
```
Comment (rosen-vladimirov) [inline knowledge-base/how-to-capture-traffic-from-another-pc.md:40]: We must note that Fiddler's root certificate must be trusted and HTTPS capturing to be enabled on the machine where Fiddler Everywhere is running. We are going to remove this limitation in the future, but we have it for the moment.

File: traffic/configure-ios.md [hunk 4]
```diff
@@ -15,28 +15,25 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
...
+1. [Inspect the traffic](#inspecting-traffic).
 
 ## Prerequisites
 
 - Install [the latest version of Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
 
 - Use an iOS device or an iOS simulator within the same local network as the Fiddler Everywhere host.
 
-## Configuring Fiddler Everywhere
+## Configuring Fiddler Everywhere Host
```
Comment (rosen-vladimirov) [inline traffic/configure-ios.md:29]: We must note that Fiddler's root certificate must be trusted and HTTPS capturing to be enabled on the machine where Fiddler Everywhere is running. We are going to remove this limitation in the future, but we have it for the moment.

**Review by rosen-vladimirov — APPROVED** [review]

File: traffic/configure-android.md (hunk 1 above)
Comment (NickIliev) [reply traffic/configure-android.md:51]: WIll crate a task for that one

**Pull request merged**